Makefile

```
@@ -25,7 +25,6 @@ presubmit-cleanup = \
 setup:
 	bash ./development/ecr/ecr-command.sh install-ecr-public
```

nit: Erring on the side of pedantry, but I think this will work just fine without `bash` here. For example, https://github.com/aws/eks-distro/blob/main/Makefile#L88
```suggestion
	development/ecr/ecr-command.sh install-ecr-public
```

---

Just following the style, but yeh, calling bash there would just result in two bashes